web/docs/understand/data-model/schemas.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_position: 1
+---
+
+# Schemas
+
+VAST defines a **schema** as a *named* record type. Since all types can have
+a name, we define a schema as a type a that (1) is a record, and (2) has a name.
+From a data perspective, the schema specifies the column structure of a
```

Comment:
```suggestion
From a data perspective, the schema specifies the structure of a
```

Given that it also defines whether a field is a list it isn't correct to just refer to columns here, at least not in the traditional sense of a 2-dimensional table. I'd just remove the word here.